source/threads.tex

```diff
@@ -3565,12 +3678,16 @@
 is undefined if the contained pointer \tcode{pm} is not null and the lockable
 object pointed to by \tcode{pm} does not exist for the entire remaining
 lifetime\iref{basic.life} of the \tcode{shared_lock} object. The supplied
-\tcode{Mutex} type shall meet the shared mutex
-requirements\iref{thread.sharedtimedmutex.requirements}.
+\tcode{Mutex} type shall meet the \oldconcept{SharedLockable}
+requirements\iref{thread.req.lockable.shared}.
 
 \pnum
 \begin{note}
-\tcode{shared_lock<Mutex>} meets the \oldconcept{TimedLockable}
```

Comment:
This edit isn't present in the paper; I assume the paper misrepresented the status quo?

Reply:
What is not present? This is point 12 in the paper.

Reply:
Ah yes, I misread the diff here (which is a bit rougher than the diff in the paper). All good. Thanks!